### DISCUSSION TITLE
ar-scale="fixed" not working for Quick Look.

### DISCUSSION THREAD

**Discussion by samsloan** [google/model-viewer#4581] [Q&A]
Hi there,

I have been trying to get ar-scale="fixed" to work in Quick Look but no matter what I try I can't get the scale to lock. The documentation says it should work across all 3 AR modes but I am not having success.

Are there any known issues or work arounds for this problem?

Cheers,
Sam

**Comment by elalish** [comment]
This could be a Safari regression - would you mind filing an issue and filling in the template completely so we can repro?
